Cargo.toml

```diff
@@ -8,6 +8,12 @@ license = "MIT/Apache-2.0"
 [dependencies]
 clap = "2.33.0"
 goblin = "0.0.23"
-symbolic-debuginfo = "6.1.3"
-pdb = "0.5.0"
-uuid = "0.7.2"
\ No newline at end of file
+is_sorted = "0.1.1"
+log = "0.4.8"
+pdb = { git = "https://github.com/willglynn/pdb", rev = "2ebb389828b349a28cf42e83c9ba5bf7bbfda6ed"}
+simplelog = "0.7.3"
+symbolic-common = "6.1.4"
+symbolic-debuginfo = "6.1.4"
+symbolic-demangle = "6.1.4"
+symbolic-minidump = "6.1.4"
```

Comment:
Are we dependent on the patch versions for all of these crates, or can we get by with just major.minor?